src/components/createEditCqlLibrary/CreateEditCqlLibrary.tsx

```diff
@@ -116,81 +117,99 @@ const CreateEditCqlLibrary = () => {
+            >
+              <FormRow tw="w-72">
+                <TextInput
+                  type="text"
+                  id="cqlLibraryName"
+                  {...formik.getFieldProps("cqlLibraryName")}
+                  placeholder="Enter a Cql Library Name"
+                  data-testid="cql-library-name-text-field"
                 >
-                  {Model[modelKey]}
-                </MenuItem>
-              );
-            })}
-          </TextField>
-        </FormControl>
-        <FormRow>
-          <Button
-            id="saveBtn"
-            buttonTitle={id ? "Update CQL Library" : "Create Cql Library"}
-            type="submit"
-            tw="mr-3"
-            data-testid="create-new-cql-library-save-button"
-            disabled={
-              !(formik.isValid && formik.dirty) ||
-              (!!id &&
-                (_.isNil(loadedCqlLibrary) || _.isNil(loadedCqlLibrary.id)))
-            }
-          />
-          <Button
-            id="cancelBtn"
-            buttonTitle="Cancel"
-            type="button"
-            variant="white"
-            onClick={() => {
-              history.push("/cql-libraries");
-            }}
-            data-testid="create-new-cql-library-cancel-button"
-          />
-        </FormRow>
-      </form>
-    </div>
+                  <Label htmlFor="cqlLibraryName" text="Cql Library Name" />
+                  {formikErrorHandler("cqlLibraryName", true)}
+                </TextInput>
+              </FormRow>
+              <FormControl tw="w-72">
+                <Label text="CQL Library Model" />
+                <TextField
+                  size="small"
+                  select
+                  InputLabelProps={{ shrink: false }}
+                  label={formik.values.model === "" ? "Select a model" : " "}
+                  id="cqlLibraryModel"
+                  data-testid="cql-library-model-select"
+                  name={"model"}
+                  {...formik.getFieldProps("model")}
+                  error={formik.touched.model && Boolean(formik.errors.model)}
+                  helperText={formik.touched.model && formik.errors.model}
+                >
+                  {Object.keys(Model).map((modelKey) => {
+                    return (
+                      <MenuItem
+                        key={modelKey}
+                        value={Model[modelKey]}
+                        data-testid={`cql-library-model-option-${Model[modelKey]}`}
+                      >
+                        {Model[modelKey]}
+                      </MenuItem>
+                    );
+                  })}
+                </TextField>
+              </FormControl>
+              <FormRow>
+                <Button
+                  id="saveBtn"
+                  buttonTitle={id ? "Update CQL Library" : "Create Cql Library"}
+                  type="submit"
+                  tw="mr-3"
+                  data-testid="create-new-cql-library-save-button"
```

Comment:
could this be `cql-library-save-button` and for cancel `cql-library-cancel-button`, because this is save+update I think.

Reply:
I agree with, I have changed the names